src/explore-education-statistics-common/src/modules/data-catalogue/components/ChangeItem.tsx

```diff
@@ -0,0 +1,84 @@
+import FilterChangeLabel from '@common/modules/data-catalogue/components/FilterChangeLabel';
+import FilterOptionChangeLabel from '@common/modules/data-catalogue/components/FilterOptionChangeLabel';
+import GeographicLevelChangeLabel from '@common/modules/data-catalogue/components/GeographicLevelChangeLabel';
+import IndicatorChangeLabel from '@common/modules/data-catalogue/components/IndicatorChangeLabel';
+import LocationGroupChangeLabel from '@common/modules/data-catalogue/components/LocationGroupChangeLabel';
+import LocationOptionChangeLabel from '@common/modules/data-catalogue/components/LocationOptionChangeLabel';
+import TimePeriodOptionChangeLabel from '@common/modules/data-catalogue/components/TimePeriodOptionChangeLabel';
+import { Change, ChangeSet } from '@common/services/types/apiDataSetChanges';
+import {
+  Filter,
+  FilterOption,
+  GeographicLevel,
+  IndicatorOption,
+  LocationGroup,
+  LocationOption,
+  TimePeriodOption,
+} from '@common/services/types/apiDataSetMeta';
+import React from 'react';
+
+export interface ChangeItemProps<TState> {
+  change: Change<TState>;
+  metaType: keyof ChangeSet;
+}
+
+export default function ChangeItem<TState>({
```

Comment:
It'd be good to have at least some basic tests for this and the other untested new components.

Reply:
I've just moved this component into `ChangeList.tsx` and added tests for this.

However, I've not added any further tests around the change label components as these are pretty thoroughly tested by the `ChangeSection` suite. There isn't really any value in testing the label components individually as they're not meant to be used in isolation, and are only in  separate files for tidiness.